### PR TITLE
remove java-android-playbook

### PR DESCRIPTION
**Pull request by bitvirago**

### CODE REVIEW

File: weekly-cache-update-playbook.yml [hunk 1]
```diff
@@ -12,6 +12,20 @@
...
+    - role: jenv
+      tags: java-android
+    - role: android
+      tags: java-android
+    - role: tools
+      tags: java-adndroid
+    - role: patch_bashrc
+      tags: java-android
+    - role: flutter
+      tags: java-adndroid
```
Comment (ghaabor) [inline weekly-cache-update-playbook.yml]: typo

File: weekly-cache-update-playbook.yml [hunk 2]
```diff
@@ -12,6 +12,20 @@
...
     - bitrise_cli_version: 1.29.0
+  roles:
+    - role: java
+      tags: java-android
+    - role: jenv
+      tags: java-android
+    - role: android
+      tags: java-android
+    - role: tools
+      tags: java-adndroid
```
Comment (ghaabor) [inline weekly-cache-update-playbook.yml]: typo

File: weekly-cache-update-playbook.yml (hunk 1 above)
Comment (bitvirago) [reply weekly-cache-update-playbook.yml]: fixed

File: weekly-cache-update-playbook.yml (hunk 2 above)
Comment (bitvirago) [reply weekly-cache-update-playbook.yml]: fixed

**Pull request merged**